src/diffusers_interpret/generated_images.py

```diff
@@ -144,3 +144,18 @@ def show(self, width: Union[str, int] = "100%", height: Union[str, int] = "400px
         self.image_slider_iframe.width = width
         self.image_slider_iframe.height = height
         display.update(self.image_slider_iframe)
+
+    def gif(self, file_name = "diffusion_process.gif", duration = 400, show = True):
+        '''
+        Generate and display a GIF from the denoising process
+        '''
+        self[0].save(file_name,
+                     save_all = True,
+                     append_images = self,

```

Comment:
In order not to have `self[0]` twice in the gif :)
```suggestion
                     append_images = self[1:],

```